osidb/filters.py

```diff
@@ -779,3 +780,16 @@ class Meta:
             # versions fields
             "versions__version": ["exact"],
         }
+
+
+class AlertFilter(IncludeFieldsFilterSet, ExcludeFieldsFilterSet):
+    owner_uuid = CharFilter(field_name="object_id")
+    owner_model = CharFilter(field_name="content_type__model")
```

Comment:
I am thinking whether the `owner_` naming is good. At least for me it sounds like something about a person which is responsible for (resolving) the alert. I am thinking how to denote the relation by other name. What would you say about eg. `parent_` ?